### ISSUE TITLE
Dependabot unsupported timezone: `"Europe/Kyiv"`

### ISSUE THREAD

**Issue by yukal** [dependabot/dependabot-core#6132]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package ecosystem

npm

### Package manager version

6.14.16, 8.11.0, 8.19.2

### Language version

12, 14, 16, 18

### Manifest location and content before the Dependabot update

There is only one package.json file exists at the root of my project
https://github.com/yukal/experimentals/blob/dev/package.json

### dependabot.yml content

https://github.com/yukal/experimentals/blob/b395ce48f2cc3a8dc83c2bbeb40da403a051a5c8/.github/dependabot.yml#L11

### What you expected to see, versus what you actually saw

I carefully read the documentation about the [timezone](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduletimezone) following the [List of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), I found the timezone I needed.
I expected that the timezone `Europe/Kyiv` should work well, but I got an error: "is not a valid timezone".

### Images of the diff or a link to the PR, issue, or logs

Here the dump of the logs
https://github.com/yukal/experimentals/runs/9449227640

**Comment by jeffwidman** [comment]
I just checked and the internal Dependabot API that handles the config file is using the timezone dataset provided by the Ruby gem: `"tzinfo-data", "1.2022.6"` https://github.com/tzinfo/tzinfo-data/releases/tag/v1.2022.6

It looks like the `Kyiv` spelling landed fairly only in August in `2022b`: https://stackoverflow.com/a/73709276/770425 but 

Surprisingly however, the Ruby gem claims to pull in the `2022f` release of the IANA database, which is definitely more recent than `2022b` where `Kyiv` was added... so it _should_ work.

I'm wondering if that Dependabot CI check uses a different set of code for validation than what the API service uses... I'll have to check internally on that one.

**Comment by jeffwidman** [comment]
So I found an internal place where we're hardcoding the list of timezones, and that's likely why this isn't getting picked up. I filed an internal ticket for myself to remove the hardcoding... it'll probably take a week or two before I get time to tackle it though.

**Comment by jeffwidman** [comment]
👋 So the hardcoded list is actually the config file schema JSON. While we don't expose that today, we may in the future (https://github.com/dependabot/dependabot-core/issues/1927) in which case we'll need that file to stay static.

So I updated the list of timezones specified in the schema to the ones listed in the IANA `2022g` release from 2022-11-29: https://data.iana.org/time-zones/tzdb/NEWS

That includes the `"Europe/Kyiv"` timezone.

I just deployed that, so it should work now. Thanks again for the heads up that we were behind.
